src/usr/bin/google_set_hostname

```diff
@@ -15,6 +15,25 @@
 
 # Deal with a new hostname assignment.
 
+retain_legacy_behavior() {
+	ggactl=$(which ggactl 2> /dev/null)
+	if [ -z "$ggactl" ]; then
+		# Agent doesn't have ggactl for communication, legacy behavior.
+		return 0
+		old_hostname=$($ggactl)
+	fi
+	# Don't retain legacy behavior if set_hostname or set_fqdn is enabled in
+	# the agent.
+	if ggactl coreplugin send '{"Command":"agent.config.getoption","Option":"Unstable.SetHostname"}' | grep '"Value":"true"' >/dev/null || ggactl coreplugin send '{"Command":"agent.config.getoption","Option":"Unstable.SetFqdn"}' | grep -i '"Value":"true"' >/dev/null; then
+		return 1
+	fi
+	return 0
+}
+
+if ! retain_legacy_behavior;then
```

Comment:
for consistency sake can you add a space between `;` and `then`? :-)

Reply:
Done.